tools/test.py

```diff
@@ -72,7 +79,10 @@ def merge_configs(cfg1, cfg2):
 def main():
     args = parse_args()
 
-    cfg = mmcv.Config.fromfile(args.config)
+    cfg = Config.fromfile(args.config)
+
+    if args.cfg_options is not None:
+        cfg.merge_from_dict(args.cfg_options)
```

Comment:
if `--cfg-options` defaults to an empty dict, can the above two lines be reduced into one line?

Reply:
set `default={}` now